drivers/platform/x86/asus-wrc.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * Asus Wireless Radio Control Driver
+ *
+ * Copyright (C) 2015 Endless Mobile Inc.
+ *
```

Comment:
I was not sure about this one either. @dsd @rshuler @blandford Do you guys know what is Endless' correct name for copyright assignment?